pytest/tests/shardnet/restake.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import shlex
+import random
+import sys
+from rc import pmap
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+import mocknet
+from configured_logger import logger
+
+
+def restart_restaked_script(restaked_url, delay_sec):
+    s = '''
+        killall restaked
+        wget '{restaked_url}' -O /home/ubuntu/restaked
```

Comment:
No need for quotes, shlex.quote will take care of it.  In fact, with quotes this is invalid.